lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/events/BasicEvent.java

```diff
@@ -30,8 +31,8 @@ public class BasicEvent implements Event {
     private LinkedHashMap<String, Event> checkIfDisabled;
     private LinkedHashMap<String, Event> checkFeatureFlag;
     private LinkedHashMap<String, Event> checkJourneyContext;
-    private List<AuditEventTypes> auditEvents;
-    private LinkedHashMap<String, String> auditContext;
+    private List<AuditEventTypes> auditEvents = new ArrayList<>();
+    private LinkedHashMap<String, String> auditContext = new LinkedHashMap<>();
```

Comment:
You're adding a default here, but also null-checking - do we need both?

If we stick with the defaults, do these need to be mutable?

Reply:
I'll revert this and update the test instead - if we were to go down the path of adding defaults I think it would need more time than we really have to do it properly.